src/routes/evento.routes.js

```diff
@@ -0,0 +1,32 @@
+const { Router } = require('express')
+const Evento = require('../models/evento')
+const verificarToken = require('../tools/verificarToken')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Backend/pullRequest?prid=7138706)
```suggestion
const verificarToken = require("../tools/verificarToken")
```